src/styles/utils/variable.scss

```diff
@@ -0,0 +1,27 @@
+// Nav
+$nav-hight: 60px;
+$nav-color: #00acdc;
+$nav-link-color: #060b35;
+
+// Card-grid
+$column-count: 1;
+$small-min-width: 488px;
+$medium-min-width: 768px;
+$large-min-width: 1024px;
+
+// Card-size
+$main-card-width: 100px;
+$percent-width-img: 0.6;
+$percent-hight-img: 0.34;
+$percent-wdith-content: 0.66;
+$width-img: $main-card-width + $main-card-width * $percent-width-img;
+$hight-img: $main-card-width + $main-card-width * $percent-hight-img;
+$width-content: $main-card-width + $main-card-width * $percent-wdith-content;
+$width-stars: ($width-content * 0.96) px;
```

Comment:
The unit `px` should not be included inside the parentheses. The correct syntax is `$width-stars: $width-content * 0.96;`.